ecosysboard/http/komodo.go

```diff
@@ -0,0 +1,53 @@
+/******************************************************************************
+ * Copyright © 2013-2019 The Komodo Platform Developers.                      *
+ *                                                                            *
+ * See the AUTHORS, DEVELOPER-AGREEMENT and LICENSE files at                  *
+ * the top-level directory of this distribution for the individual copyright  *
+ * holder information and the developer policies on copyright and licensing.  *
+ *                                                                            *
+ * Unless otherwise agreed in a custom licensing agreement, no part of the    *
+ * Komodo Platform software, including this file may be copied, modified,     *
+ * propagated or distributed except according to the terms contained in the   *
+ * LICENSE file                                                               *
+ *                                                                            *
+ * Removal or modification of this copyright notice is prohibited.            *
+ *                                                                            *
+ ******************************************************************************/
+
+package http
+
+import (
+	"encoding/json"
+	"github.com/kpango/glg"
+	"github.com/valyala/fasthttp"
+	"net/http"
+	"strings"
+)
+
+var komodoCoinsToCoinpaprikaRegistry = map[string]string{
```

Comment:
it probably makes sense to import the coins from a `esb_config.json` file please

Reply:
yes i will do it